tests/PHPTestCase.php

```diff
@@ -56,9 +59,18 @@ public function caseProvider()
      * @param string $token
      * @return \PSX\Sandbox\Runtime
      */
-    protected function newRuntime($token)
+    protected function newRuntime(string $token, ?array $options) : \PSX\Sandbox\Runtime
     {
-        $runtime = new Runtime($token, null, __DIR__ . '/cache');
+        $securityManager = new SecurityManager();
```

Comment:
if we have the `Configuration` class we could add a `Configuration::fromArray($options)` named constructor and pass it to the `SecurityManager`